ci/check-in.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -eu
+
+# This is not a very smart script
+if [ $# != 2 ]; then
+  echo "usage: $0 <since> <number-of-prs-merged>"
+  if [ $# = 1 ] ; then
+    echo "help: you can find the number of PRs merged at" \
+      "https://github.com/rust-lang/rustc-dev-guide/pulls?q=is%3Apr+is%3Aclosed+updated%3A%3E$1"
+  fi
+  exit 1
+fi
+
+curl() {
+  command curl -s "$@"
+}
+
+# Get recently updated PRs
+curl "https://api.github.com/repos/rust-lang/rustc-dev-guide/pulls?state=closed&per_page=$2" \
```

Comment:
We want only merged ones, right?

```suggestion
curl "https://api.github.com/repos/rust-lang/rustc-dev-guide/pulls?state=merged&per_page=$2" \
```

Reply:
`merged` is not one of the available filters in the API; it will actually return open PRs instead. https://developer.github.com/v3/pulls/#parameters

Reply:
Oops, I missed that this from the API not the web interface :)